x-pack/legacy/plugins/ml/public/application/components/influencers_list/_influencers_list.scss

```diff
@@ -1,6 +1,12 @@
 .ml-influencers-list {
   line-height: 1.45; // SASSTODO: Calc proper value
 
+  .influencer-title {
+    font-size: $euiFontSizeM - 2px;
+    font-weight: 600;
+    line-height: 24px;
+  }
```

Comment:
Is it possible at all to use one of the pre-existing title sizing without overrides? We've created the title scale with the intention that we'd have consistency throughout all the apps in Kibana. I'm sure one of these sizes should fit this use case. http://eui.elastic.co/#/display/title

Reply:
@cchaos good spot - I have managed to remove this override, plus another one I was using on the data visualizer cards in https://github.com/elastic/kibana/pull/56741/commits/6f1de6bcaaa3b8ea4495a13b90204e054fd12013. 

I have decided to leave some `panel-title` overrides for headings on the Anomaly Explorer and Single Metric Viewer for now, to retain the current size (18px) and color. Note @mdefazio is currently looking at some options for headings and controls on these two pages, so we may be able to clean these up in a future PR.